addons/zeus/stringtable.xml

```diff
@@ -355,7 +355,7 @@
         </Key>
         <Key ID="STR_ACE_Zeus_ModuleEditableObjects_AllCurators_Tooltip">
             <English>Apply changes to all curators</English>
-            <German>Änderungen bei allen Zeus' aktualisieren</German>
+            <German>Änderungen bei allen Zeusen aktualisieren</German>
```

Comment:
Was zum Fick. Auch wenn die linguistische Wurzel einfach nur "Gott" bedeutet, is das im deutschen Sprachgebrauch für allem der Name einer (fiktiven) Person. Die englischen Einträge in den Vanilla-Menüs benutzen glaub ich alle "curator", vielleicht wäre das ein Ansatz.